cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/AbstractConnectOperator.java

```diff
@@ -356,11 +356,11 @@ private JsonObject asJson(KafkaConnectorSpec spec) {
                         }
                     }
                 } else {
-                    log.error("{}: Current Kafka Connect resource not found", reconciliation);
-                    updateStatusPromise.fail("Current Kafka Connect resource not found");
+                    log.error("{}: Current {} resource not found", reconciliation, resource.getClass().getTypeName());
```

Comment:
You should use `resource.getKind()` instead of `resource.getClass().getTypeName()` I think.

Reply:
fair enough. done.